test/extended/olm/olmv1.go

```diff
@@ -419,3 +500,89 @@ func checkFeatureCapability(ctx context.Context, oc *exutil.CLI) {
 		g.Skip("Test only runs with OperatorLifecycleManagerV1 capability")
 	}
 }
+
+// verifyAPIEndpoint runs a job to validate the given service endpoint of a ClusterCatalog
+func verifyAPIEndpoint(ctx g.SpecContext, oc *exutil.CLI, namespace, catalogName, endpoint string) {
+	jobName := fmt.Sprintf("test-catalog-%s-%s-%s", catalogName, endpoint, rand.String(5))
```

Comment:
I see `metas?schema=olm.package` being passed as endpoint and then used in the job name. Seems like that's an invalid name? Maybe pass `url.Params` as a separate argument in the function?